coherent_rates/plot.py

```diff
@@ -507,3 +513,111 @@ def plot_scattered_energy_change_comparison(
     fig.legend()
     fig.show()
     input()
+
+
+def plot_scattered_energy_change_state(
+    system: PeriodicSystem,
+    config: PeriodicSystemConfig,
+    state: StateVector[Any],
+    *,
+    nk_points: list[tuple[int, ...]] | None = None,
+) -> None:
+    fig, ax = get_figure(None)
+
+    bound_data = get_scattered_energy_change_against_k(
+        system,
+        config,
+        state,
+        nk_points=nk_points,
+    )
+
+    k_points = bound_data["basis"].k_points
+    k_points_squared = np.square(k_points)
```

Comment:
add a plot_value_list_against_momentum and plot_value_list_against_momentum_squared